onnx_caffe2/backend.py

```diff
@@ -472,6 +473,12 @@ def _direct_initialize_inputs(cls, inputs, initialized, ws, device_option):
             shape = list(d.dim_value for d in value_info.type.tensor_type.shape.dim)
             ws.FeedBlob(value_info.name, np.ones(shape), device_option)
 
+    @staticmethod
+    def optimize_onnx(input):
+        proc = Popen(['optimize-onnx'], stdin=PIPE, stdout=PIPE, stderr=PIPE)
```

Comment:
should it be a relative path?

Reply:
I've changed setup.py to put `optimize-onnx` in the bin/ of the python install/virtualenv, so it should be fine to look for it in PATH

Reply:
@anderspapitto This is not always true, you can use the virtualenv without activating the virtualenv (thus `virtualenv/bin` is not in your `$PATH`).